app/sessions/session-details/session-details.component.html

```diff
@@ -19,15 +19,13 @@
 				<StackLayout class="session-title-container" margin="16 16 16 0" row="1" col="1">
 					<Label [text]="session.title" textWrap="true" class="session-page-title"></Label>
 					<Label [text]="session.range + ', ' + session.room + ', '" textWrap="true" class="session-page-subtitle"></Label>
-					<!-- Need to add floorplans as session.floor for custom floor rendering -->
```

Comment:
Was this feature added?

Reply:
Yes. Added in latest commit

Reply:
Can you add this to the excel sheet for tracking?